src/electionguard_cli/cli_steps/election_builder_step.py

```diff
@@ -28,13 +29,15 @@ def _build_election(
     ) -> BuildElectionResults:
         self.print_header("Building election")
 
+        click.echo("Initializing public key and commitment hash")
```

Comment:
For some reason the ElectionBuilder constructor takes a very long time to run on my machine, and the build command not so speedy either, so added some text so the user isn't left wondering.